rust-app/Cargo.toml

```diff
@@ -27,12 +27,14 @@ speculos = [ "ledger_device_sdk/speculos", "ledger-log/speculos", "ledger-log/lo
 extra_debug = ["ledger-log/log_trace"]
 
 [target.'cfg(target_family = "bolos")'.dependencies]
-ledger_device_sdk = "1.7.1"
-ledger_secure_sdk_sys = "1.2.0"
+ledger_device_sdk = "1.18.4"
+ledger_secure_sdk_sys = "1.5.3"
```

Comment:
Could you please update to last versions:
- ledger_device_sdk = `1.19.1`
- ledger_secure_sdk_sys = `1.6.1`
Thank you 🙏 !

Reply:
Fixed